include/RED4ext/Dump/Reflection-inl.hpp

```diff
@@ -41,16 +41,30 @@ RED4EXT_INLINE void Dump(std::filesystem::path aOutPath, std::filesystem::path a
     {
         size_t i = 0;
 
-        // Special case for AI
-        if (aInput.size() >= 2 && aInput[0] == 'A' && aInput[1] == 'I')
-        {
-            i = 2;
-        }
+        static constexpr std::pair<std::string_view, bool> uniqueNamespaces[] = {
+            {"AI", false}, {"inGame", true}, {"GpuWrapApi", true}, {"GpuWrapApiVertexPacking", true}};
 
-        // Special case of "in", this will break directory layout for "ink", "interop", etc..
-        if (aInput.starts_with("inGame"))
+        for (const auto& [name, isSpecialCase] : uniqueNamespaces)
         {
-            return "";
+            if (aInput.size() > name.size() && aInput.starts_with(name))
+            {
+                i = name.size();
+
+                if (isSpecialCase)
+                {
+                    // Special case of "in", this will break directory layout for "ink", "interop", etc..
+                    if (aInput.starts_with("inGame"))
+                    {
+                        return "";
+                    }
+
+                    // Special case for enums under `GpuWrapApi` i.e. `eTextureType`
+                    if (name.starts_with("GpuWrapApi") && aInput[i] == 'e')
+                    {
+                        return aInput.substr(0, i);
+                    }
```

Comment:
This code here is "smelly". You already define `inGame` and `GpuWrapApi` as special, then do some special handling. I would say that `inGame` should be in a separate if, not in this for and this `isSpecialCase` block should just return `substr(...)`.